tests/apps/actordotnet/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/aspnet:3.1 AS base
+WORKDIR /app
+EXPOSE 3000
+
+FROM mcr.microsoft.com/dotnet/sdk:3.1 AS build
+WORKDIR /src
+COPY ["CarActor.csproj", "./"]
+RUN dotnet restore "CarActor.csproj"
+COPY . .
+WORKDIR "/src/."
+RUN dotnet build "CarActor.csproj" -c Release -o /app/build
+
+FROM build AS publish
+RUN dotnet publish "CarActor.csproj" -c Release -o /app/publish

```

Comment:
not sure where you copied this dockerfile from, but it's way more complex than necessary. `dotnet publish` does a `dotnet build` so you don't need this layer.

Reply:
It was autogenerated by VS code. I can simplify it.

Reply:
huh, wierd. @philliphoff is this you? I haven't seen this pattern before 

Let's not derail this PR blocking on this either way.

Reply:
@rynowak This is the standard "general Dockerfile for .NET Core build/debug/publish" used across the container-related tooling. For single or other very specific use cases, it's admittedly going to be overly complex.